邓堂瑞作业

CZ-dd opened the pull request:
试试能不能pull上去, level还没做完

CZ-dd committed:
试试

这咋搞

CZ-dd committed:
Update p02 main.c

修改了之前跑p04数据的更改

CZ-dd committed:
作业

无

CZ-dd committed:
作业

试试能不能pull上去

CZ-dd committed:
Merge branch 'main' of https://github.com/CZ-dd/c2023-challenge

CZ-dd committed:
作业

为什么始终不行???

CZ-dd committed:
Merge branch 'main' of https://github.com/CZ-dd/c2023-challenge

CZ-dd committed:
level9和level10

大致完成了level 9和level 10, 更新了level 2, 分为了多个文件

CZ-dd committed:
修复了bug

p08中, 分数不会低于0
p10中, 完善了delete(外接固态掉盘, 之前写的没保存, 重写了)

CZ-dd committed:
更新了p03, p05

p03添加了计算耗时
p05支持输入字符串进行加解密

luckymark commented on level1/CMakeLists.txt:
```diff
@@ -16,6 +17,14 @@ add_executable(p07_maze p07_maze/main.c)
-add_executable(p10_warehouse p10_warehouse/main.c)
\ No newline at end of file
+add_executable(p09_linked_list p09_linked_list/main.c
+        p09_linked_list/link.h
```
头文件不用放到cmakelists里，因为编译是以点C源文件为单位的，头文件会被include进去

luckymark commented on level1/cmake-build-debug/.cmake/api/v1/reply/codemodel-v2-85747c7dbf81af576a00.json:
cmake-build-debug这类生成的文件，一般不提交到git中，移除的办法可以到网上搜索一下

CZ-dd committed:
level 2 GA & level 1

修复了level 1 p04中的一个小问题, 大致完成了GA, 将迷宫生成单独存放在一个文件并封装 , 将生成迷宫所用的栈封装,  使用遗传算法寻找迷宫的解, 为方便调试, 随机数种子固定.
但GA中仍存在大量的问题, 能力不足, 暂且搁置
